tex/chapters/Results.tex

```diff
@@ -2,12 +2,221 @@
 
 \chapter{Results}\label{cha:results}
 
-\section{Introduction}
+In this chapter, the implementation of a fingerprint function within the Axelrod-Python library will be examined.
+This includes the addition of two strategy transformers, the Dual and JossAnn as defined in \ref{}. %TODO
+Then several results will be presented where analytical fingerprints are compared with analytical ones.
```

Comment:
You should also have a section discusses the development process (the PR discussion etc in particular).